pkg/tsdb/influxdb/flux/flux.go

```diff
@@ -22,31 +21,29 @@ func init() {
 }
 
 // Query builds flux queries, executes them, and returns the results.
-//nolint: staticcheck // plugins.DataQuery deprecated
-func Query(ctx context.Context, httpClientProvider httpclient.Provider, dsInfo *models.DataSource, tsdbQuery plugins.DataQuery) (
-	plugins.DataResponse, error) {
+func Query(ctx context.Context, httpClientProvider httpclient.Provider, dsInfo *models.DatasourceInfo, tsdbQuery backend.QueryDataRequest) (
```

Comment:
`httpClientProvider` doesn't seem to be needed any longer since you use `dsInfo.HTTPClient` instead.